exe/wallet/Main.hs

```diff
@@ -173,20 +182,34 @@ exec manager args
             (ApiT wPwd)
 
     | args `isPresent` command "wallet" && args `isPresent` command "update" = do
-        wId <- args `parseArg` longOption "id"
+        wId <- args `parseArg` longOption "wallet-id"
         wName <- args `parseArg` longOption "name"
         runClient $ putWallet (ApiT wId) $ WalletPutData
             (Just $ ApiT wName)
 
     | args `isPresent` command "wallet" && args `isPresent` command "delete" = do
-        wId <- args `parseArg` longOption "id"
+        wId <- args `parseArg` longOption "wallet-id"
         runClient $ void $ deleteWallet (ApiT wId)
 
+    | args `isPresent` command "transaction" && args `isPresent` command "create" = do
+        wId <- args `parseArg` longOption "wallet-id"
+        ts <- args `parseAllArgs` longOption "target"
+        (wPwd, _) <- do
+            let prompt = "Please enter a passphrase: "
+            let parser = fromText @(Passphrase "encryption")
+            getSensitiveLine prompt Nothing parser
+        runClient $ createTransaction (ApiT wId) $ PostTransactionData
+            -- NOTE: we are sure this will be non-empty
+            (fromList ts)
+            (ApiT wPwd)
```

Comment:
It should probably print something on success (e.g. a transaction ID), and on failure.

Reply:
I think it does already? cf `runClient` https://github.com/input-output-hk/cardano-wallet/blob/master/exe/wallet/Main.hs#L204-L217